src/bridge/BridgeSettlerBase.sol

```diff
@@ -58,7 +58,8 @@ abstract contract BridgeSettlerBase is Basic {
             // It is not possible to call it directly as the taker is going to be the BridgeSettler
             // instead of the user, so, user assets needs to be pulled to BridgeSettler before
             // attempting to do this swap.
-            // Security risks are inherited from Settler
+            // Settler can take over the assets if settlerData starts with TRANSFER_FROM which
+            // can put them at risk if AllowedSlippage is not properly set.
```

Comment:
It's not just `TRANSFER_FROM`; it's any VIP action. And the risk here is specifically sandwiching from MEV attacks and forcing the swap to its slippage limit. Consider stating this explicitly.